src/main/java/fr/quatrevieux/araknemu/core/config/PoolUtils.java

```diff
@@ -172,6 +173,31 @@ public double decimal(String key) {
         return decimal(key, 0d);
     }
 
+    /**
+     * Parse a config item as a ZoneId
+     * @param key
+     * @param defaultValue
+     * @return
+     */
+    public ZoneId zoneId(String key, ZoneId defaultValue) {
+        final String value = pool.get(key);
+
+        return value != null
+            ? ZoneId.of(value)
+            : defaultValue
+        ;
+    }
+
+    /**
+     * Parse a config item as a ZoneId
+     * If the item is not configured, will return Europe/Paris
+     * @param key
+     * @param defaultValue
+     * @return
+     */
+    public ZoneId zoneId(String key) {
+        return zoneId(key, ZoneId.of("Europe/Paris"));
+    }
```

Comment:
Can you add unit test for both methods on PoolUtilsTest ?

Reply:
Fixed in commit [#3e74dfb](https://github.com/Arakne/Araknemu/pull/309/commits/3e74dfbf70e1bf06aa444c1dd4f9d4362f5936cc)